src/docs/orgs.md

```diff
@@ -0,0 +1,59 @@
+# Pants for Organisations
+
+Pants works well for organizations with a large number of developers working in a multi-tenant repo.
+Typically, repos becomes slower over time as more source and test files are added because most
+build systems build and test everything every time. Pants enables the benefits of a single repo
+with the speed of per-project repos by only operating on repo "slices".
+
+## Frontend & Backend Example
+
+Imagine a simple repo that contains both a frontend and backend service, as illustrated by the
+following dependency graph.
+
+![orgs](images/orgs.png)
+
+Consider a frontend developer that only works on the frontend service. With most build systems, the
+developer would need to compile everything every time, slowing them down over time as more and more
+code is added to repo. With Pants, the frontend developer only needs to compile the code directly
+used by their service. For example, consider building `org.pantsbuild.fe.service`. Regardless of
+how complex the backend code grows over time, the frontend build will not slow down.
+
+![orgs_fe](images/orgs_fe.png)
+
+Now consider a backend developer who only works on the backend service. When building the backend
+service, only the code required by the backend service is built, regardless of what other code
+is in the repo. The ability to build "slices" of the repo fundamental to enabling multiple users
+and teams to work together in a single repo. Note how only the backend code would be build when
+building `org.pantsbuild.be.service`:
+
+![orgs_be](images/orgs_be.png)
+
+## Continuous Integration
+
+Most organizations require test pass before merging a change to master. Pants enables fast CI by
+only running tests _affected_ by a change. That is, only targets that transitively depend on a
+changed target have their tests run. In practice, a change to `README.md` might not trigger *any*
+tests, a change to a leaf-node in the dependency graph would only trigger it's direct tests, and
```

Comment:
`leaf-node` might be a little overloaded these days because people have different conventions. How about 
```
top-level node such as `org.pantsbuild.fe.service`
``` 
to keep the language consistent with `low-level` node?